cavern/src/intTest/java/org/opencadc/cavern/TransferTest.java

```diff
@@ -95,15 +107,20 @@ public class TransferTest extends org.opencadc.conformance.vos.TransferTest {
 
     static {
         Log4jInit.setLevel("org.opencadc.conformance.vos", Level.INFO);
-        
         Log4jInit.setLevel("org.opencadc.vospace", Level.INFO);
         Log4jInit.setLevel("org.opencadc.cavern", Level.INFO);
     }
     
     public TransferTest() {
         super(Constants.RESOURCE_ID, Constants.TEST_CERT);
+
```

Comment:
check cavern NodesTest setup and move things to Constants for consistency

Reply:
Updated.